include/common.php

```diff
@@ -1591,14 +1601,19 @@ function get_cdash_dashboard_xml($projectname, $date)
         $xml .= add_XML_value('id', $userid);
 
         // Is the user super administrator
-        $userquery = pdo_query('SELECT admin FROM ' . qid('user') . " WHERE id='$userid'");
-        $user_array = pdo_fetch_array($userquery);
-        $xml .= add_XML_value('admin', $user_array[0]);
+        require_once 'models/user.php';
+        $user = new User();
+        $user->Id = $userid;
+        $user->Fill();
+        $xml .= add_XML_value('admin', $user->Admin);
```

Comment:
I used `Fill()` instead here because `IsAdmin()` returns true or false, while the `admin` column in the database (and thus `->Admin`) is either 1 or 0.

I know PHP is loosely typed so this distinction probably doesn't matter there, but I'm concerned we might have some explicit checks for `admin == 1` lurking in XSL land.